storey/flow.py

```diff
@@ -1565,8 +1721,6 @@ class ParallelExecution(Flow):
     def __init__(
         self,
         runnables: list[ParallelExecutionRunnable],
-        max_processes: Optional[int] = None,
-        max_threads: Optional[int] = None,
         **kwargs,
     ):
         super().__init__(**kwargs)
```

Comment:
When introducing multiple inheritance, a single super init is no longer sufficient. But as mentioned elsewhere, I'm in favour of using composition instead.

Reply:
I added a `super().__init__(**kwargs)` in the `Executor` so it is OK.